rpc/src/main/proto/distkv_pb.proto

```diff
@@ -26,59 +26,50 @@ enum RequestType {
 
     DEACTIVE_NAMESPACE = 3;
 
+    DROP = 4;
+
+    EXPIRE = 5;
+
     // string concept
     STR_PUT = 101;
     STR_GET = 102;
-    STR_DROP = 103;
 
     // list concept
     LIST_PUT = 201;
     LIST_GET = 202;
-    LIST_DROP = 203;
-    LIST_LPUT = 204;
-    LIST_RPUT = 205;
-    LIST_REMOVE = 206;
-    LIST_MREMOVE = 207;
+    LIST_LPUT = 203;
+    LIST_RPUT = 204;
+    LIST_REMOVE = 205;
+    LIST_MREMOVE = 206;
 
     // set concept
     SET_PUT = 301;
     SET_GET = 302;
-    SET_DROP = 303;
     SET_PUT_ITEM = 304;
     SET_REMOVE_ITEM = 305;
     SET_EXISTS = 306;
 
     // dict concept
     DICT_PUT = 401;
     DICT_GET = 402;
-    DICT_DROP = 403;
-    DICT_PUT_ITEM = 404;
-    DICT_GET_ITEM = 405;
-    DICT_POP_ITEM = 406;
-    DICT_REMOVE_ITEM = 407;
+    DICT_PUT_ITEM = 403;
+    DICT_GET_ITEM = 404;
+    DICT_POP_ITEM = 405;
```

Comment:
Modify only one line here。

Reply:
Because the drop number is in the middle, if it is removed, the rearrangement of the serial number is reasonable.